packages/calcite-design-tokens/support/tests/__snapshots__/index.spec.ts.snap

```diff
@@ -995,41 +1035,41 @@ export const calciteTypographyBoldMinus2h = {"fontWeight":"600","fontSize":"12px
-export const calciteTypographyWrapMediumMinus2 = {"fontWeight":"500","lineHeight":"137.5%","fontSize":"12px"};
-export const calciteTypographyWrapMediumMinus1 = {"fontWeight":"500","lineHeight":"137.5%"};
-export const calciteTypographyWrapBold0 = {"fontWeight":"600","lineHeight":"137.5%","fontSize":"16px"};
-export const calciteTypographyWrapBold1 = {"fontWeight":"600","lineHeight":"137.5%","fontSize":"18px"};
-export const calciteTypographyWrapBold2 = {"fontWeight":"600","lineHeight":"137.5%","fontSize":"20px"};
-export const calciteTypographyWrapBold3 = {"fontWeight":"600","lineHeight":"137.5%","fontSize":"24px"};
-export const calciteTypographyWrapBoldMinus2 = {"fontWeight":"600","lineHeight":"137.5%","fontSize":"12px"};
-export const calciteTypographyWrapBoldMinus1 = {"fontWeight":"600","lineHeight":"137.5%"};
-export const calciteTypographyHierarchyDisplay1 = {"fontWeight":"600","lineHeight":"137.5%","fontSize":"24px"};
-export const calciteTypographyHierarchyDisplay2 = {"fontWeight":"600","lineHeight":"137.5%","fontSize":"20px"};
-export const calciteTypographyHierarchyHeading1 = {"fontWeight":"500","lineHeight":"137.5%","fontSize":"24px"};
-export const calciteTypographyHierarchyHeading2 = {"fontWeight":"500","lineHeight":"137.5%","fontSize":"20px"};
-export const calciteTypographyHierarchyHeading3 = {"fontWeight":"500","lineHeight":"137.5%","fontSize":"18px"};
-export const calciteTypographyHierarchyHeading4 = {"fontWeight":"500","lineHeight":"137.5%","fontSize":"16px"};
-export const calciteTypographyHierarchyHeading5 = {"fontWeight":"500","lineHeight":"137.5%"};
-export const calciteTypographyHierarchyBodySnug = {"lineHeight":"137.5%"};
+export const calciteTypographyWrapLight0 = {"fontWeight":"300","lineHeight":"1.375","fontSize":"16px"};
+export const calciteTypographyWrapLight1 = {"fontWeight":"300","lineHeight":"1.375","fontSize":"18px"};
+export const calciteTypographyWrapLight2 = {"fontWeight":"300","lineHeight":"1.375","fontSize":"20px"};
+export const calciteTypographyWrapLight3 = {"fontWeight":"300","lineHeight":"1.25","fontSize":"24px"};
+export const calciteTypographyWrapLightMinus2 = {"fontWeight":"300","lineHeight":"1.375","fontSize":"12px"};
+export const calciteTypographyWrapLightMinus1 = {"fontWeight":"300","lineHeight":"1.375"};
+export const calciteTypographyWrapRegular0 = {"lineHeight":"1.375","fontSize":"16px"};
+export const calciteTypographyWrapRegular1 = {"lineHeight":"1.375","fontSize":"18px"};
+export const calciteTypographyWrapRegular2 = {"lineHeight":"1.375","fontSize":"20px"};
+export const calciteTypographyWrapRegular3 = {"lineHeight":"1.375","fontSize":"24px"};
+export const calciteTypographyWrapRegularMinus2 = {"lineHeight":"1.375","fontSize":"12px"};
+export const calciteTypographyWrapRegularMinus1 = {"lineHeight":"1.375"};
+export const calciteTypographyWrapMedium0 = {"fontWeight":"500","lineHeight":"1.375","fontSize":"16px"};
```

Comment:
Can we add a note in the PR description about line-height now being unitless?

Reply:
this was always the case

Reply:
The es6 and spec tests say otherwise. Please add a note about this change.